Snowplow/Internal/Tracker/SPTracker.m

```diff
@@ -257,8 +259,17 @@ - (void)checkInstall {
 }
 
 - (void) setTrackerData {
```

Comment:
This is clever! Sanitizing the string this way should ensure we always get easy to query `v_tracker` values 👍